packages/ui/dropdown/src/dropdown-menu-item.tsx

```diff
@@ -21,8 +22,13 @@ const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
 	isDisabled,
 	onClick,
 	variation = "",
+	isChecked = false,
```

Comment:
now as we have `_checkboxProps` prop specifically, we can pass isChecked with this, so you can remove this